src/player_activity.cpp

```diff
@@ -137,13 +138,13 @@ void player_activity::do_turn( player *p )
             break;
         case ACT_PICKAXE:
             // Based on speed, not time
-            moves_left -= p->moves;
+            moves_left -= (int)(p->moves * season_mult);
```

Comment:
most of its use cases deal with disambiguation and class related items, but iirc it's more C++ proper. may want to ask BevapDin.